File: src/main/java/me/pljr/pljrapispigot/objects/PLJRActionBar.java
package me.pljr.pljrapispigot.objects;

import me.pljr.pljrapispigot.utils.ActionBarUtil;
import org.bukkit.entity.Player;

public class PLJRActionBar {
    private final String message;
    private final long duration;

    public PLJRActionBar(String message, long duration){
        this.message = message;
        this.duration = duration;
    }

    public String getMessage() {
        return message;
    }
    public long getDuration() {
        return duration;
    }

    public void send(Player player){
        ActionBarUtil.send(player, this);
    }

    public void broadcast(){
        ActionBarUtil.broadcast(this);
    }
}

File: src/main/kotlin/me/pljr/pljrapispigot/actionbar/PLJRActionBar.kt
package me.pljr.pljrapispigot.actionbar

import org.bukkit.entity.Player

class PLJRActionBar(val message: String, val duration: Long) {

    fun send(player: Player) {
        sendActionBar(player, this)
    }

    fun broadcast() {
        broadcastActionBar(this)
    }
}